hexrd/ui/overlays/powder_diffraction.py

```diff
@@ -139,7 +139,8 @@ def generate_ring_points(self, tths, etas, panel, display_mode):
 
             if display_mode == ViewType.polar:
                 # !!! apply offset correction
-                ang_crds, _ = panel.cart_to_angles(xys, self.instrument.tvec)
+                ang_crds, _ = panel.cart_to_angles(xys,
+                                                   tvec_c=self.instrument.tvec)
```

Comment:
This is not exactly correct, but more correct than it was.  There will be other changes here soon.